cmd/enduro-a3m-worker/main.go

```diff
@@ -127,7 +127,7 @@ func main() {
 			MaxConcurrentSessionExecutionSize:  1000,
 			MaxConcurrentActivityExecutionSize: 1,
 		}
-		w := temporalsdk_worker.New(temporalClient, cfg.A3m.TaskQueue, workerOpts)
+		w := temporalsdk_worker.New(temporalClient, cfg.Pres.TaskQueue, workerOpts)
```

Comment:
This worker should always pull from the "a3m" taskqueue.  The `enduro-am-worker` should always pull from the "AM" taskqueue.

Reply:
Or we should get rid of `enduro-am-worker` altogether and rename this `enduro-preservation-worker` (or similar).

Reply:
I am not sure that renaming enduro-am-worker makes sense. there is no longer a a3m taskqueue. There is only a preservation taskqueue. Perhaps hard coding this since this should only use "a3m" would be a solution here. But I think this is mostly out of bounds with this PR. Doing it this way changes no functionality in the worker. A problem I do see if you want to use multipe types of workers (an am and a3m at the same time).